lib/jxl/enc_xyb.cc

```diff
@@ -86,13 +86,12 @@ void TestCubeRoot() {
 
     // All lanes are same
     for (size_t i = 1; i < Lanes(d); ++i) {
-      JXL_ASSERT(std::abs(approx[0] - approx[i]) <= 1.2E-7f);
+      JXL_ASSERT(std::abs(approx[0] - approx[i]) <= 5E-7f);
```

Comment:
Maybe I'm missing something here, but shouldn't this always be 0? the input to CubeRootAndAdd is the same for all lanes, and I don't see any lane crossing operation in CubeRootAndAdd -- it seems rather fishy to me if this fails (possibly indicative of a bug in hwy?)

For that matter, even the version with 1.2E-7f as an error bound seems suspicious.

Reply:
I can confirm that even before the tests would not have passed with `0` instead of `1.2E-7f`.

Reply:
That seems very surprising to me. Do you know on which targets it happens?

@jan-wassenberg do you have an idea why this might be happening?